spring-boot-actuator/pom.xml

```diff
@@ -138,6 +138,11 @@
 			<optional>true</optional>
 		</dependency>
 		<dependency>
+			<groupId>org.hibernate</groupId>
+			<artifactId>hibernate-entitymanager</artifactId>
```

Comment:
since we are intending to include in spring boot 2.0 and spring boot 2.0 supports 5.2+ , Since 5.2 entitymanager module is integrated with core module. this dependency is not necessary